crates/daphne-server/src/router/leader.rs

```diff
@@ -46,7 +47,7 @@ where
         version = ?req.version,
     )
 )]
-async fn upload<A>(
+async fn upload_13<A>(
```

Comment:
Align with notation elsewhere (we write "draft09" instead of "09")
```suggestion
async fn upload_draft13<A>(
```